app/controllers/bookings_controller.rb

```diff
@@ -9,26 +9,37 @@ def new
     @activity = Activity.find(params[:activity_id])
   end
 
+  def show
+    @activity = Activity.find(params[:activity_id])
+    @booking = Booking.find(params[:id])
+  end
+
   def create
     @booking = Booking.new(booking_params)
     @activity = Activity.find(params[:activity_id])
     @booking.activity = @activity
     @booking.user = current_user
     @booking.status = "Not confirmed"
     if @booking.save
-      redirect_to activity_path(@activity)
+      redirect_to activity_booking_path(@activity, @booking)
     else
       render :new
     end
   end
 
-  # def show
-  #   @booking = Booking.find(params[:id])
-  # end
+  def edit
+    @booking = Booking.find(params[:id])
+  end
+
+  def update
+    @booking = Booking.find(params[:id])
+    @booking.update(booking_params)
+    redirect_to activities_path
```

Comment:
I don't want to link it to the activities page, I just tried different paths to check because any path I try, it always goes to the activity page. I want to redirect to activity_booking_path(@activity, @booking), but only redirect to the activity. Can you please help? Thanks